baseplate/__init__.py

```diff
@@ -704,7 +704,7 @@ def make_child(
                 context_copy,
                 self.baseplate,
             )
-        context_copy.trace = span
+        context_copy.span = span
```

Comment:
Do you want to update the comment on L718?
> during requests as the ``trace`` attribute.

Reply:
thanks!